plugins/android-extensions/android-extensions-compiler/test/org/jetbrains/kotlin/android/parcel/AbstractParcelBytecodeListingTest.kt

```diff
@@ -20,6 +20,6 @@ abstract class AbstractParcelBytecodeListingTest : AbstractAsmLikeInstructionLis
             ?: throw RuntimeException(
                 "Unable to get a valid path from 'ideaSdk.androidPlugin.path' property, please point it to the Idea android plugin location"
             )
-        environment.updateClasspath(listOf(JvmClasspathRoot(File(androidPluginPath, "layoutlib.jar"))))
+        environment.updateClasspath(listOf(JvmClasspathRoot(File(androidPluginPath, "layoutlib-26.5.0.2.jar"))))
```

Comment:
I'd extract this file name as a constant somewhere

Reply:
Done. Moved the whole thing to the companion of `AbstractParcelBoxTest`, since the path is effectively static.